x-pack/packages/ml/agg_utils/src/build_random_sampler_aggregation.ts

```diff
@@ -24,6 +26,7 @@ export function buildRandomSamplerAggregation(
       // @ts-expect-error `random_sampler` is not yet part of `AggregationsAggregationContainer`
       random_sampler: {
         probability: sampleProbability,
+        seed: RANDOM_SAMPLER_SEED,
```

Comment:
Would be good to figure out if there's any unintended issue with using a globally static constant for seed as different searches can happen at the same time and might invalidate the cache (?). If there's no issue, a comment noting that would be great here.

Reply:
I added a comment in [6652b49](https://github.com/elastic/kibana/pull/145088/commits/6652b49c435f1a294dd4f4a7bbaa5bed13f8afcd) about this being scoped for the technical preview of Explain Log Rate Spikes. We will investigate possible updates in 8.7+.